plugins/inputs/tail/tail.go

```diff
@@ -236,6 +236,10 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 
 				if err := tailer.Err(); err != nil {
 					t.Log.Errorf("Tailing %q: %s", tailer.Filename, err.Error())
+					if strings.HasSuffix(err.Error(), "permission denied") {
```

Comment:
Have you tried to use `errors.Is(err, os.ErrPermission)` here? It might nor work as the `tail` package might not wrap the errors correctly...